README.md

```diff
@@ -1,3 +1,9 @@
+# ⚠️ This package is outdated and no longer maintained. ⚠️
+
+This package contains tests between `{Kubo,js-ipfs} <-> {Kubo,js-ipfs}`.
+
+[Js-ipfs is no longer being maintained](https://github.com/ipfs/js-ipfs/issues/4336) and [Kubo now uses Helia's interop tests](https://github.com/ipfs/kubo/issues/10013).
```

Comment:
```suggestion
[Js-ipfs is no longer being maintained](https://github.com/ipfs/js-ipfs/issues/4336) and therefore Kubo <-> Helia [interop](https://github.com/ipfs/helia/tree/main/packages/interop) tests are used instead both within Helia and [Kubo](https://github.com/ipfs/kubo/issues/10013) CI testing.
```

Your seems fine, this might be more precise though (I don't like the reference to code that might move, but that's probably better than a git commit in this instance).

Reply:
```suggestion
[Js-ipfs is no longer being maintained](https://github.com/ipfs/js-ipfs/issues/4336) and therefore Kubo <-> Helia [interop](https://github.com/ipfs/helia/tree/main/packages/interop) tests are used instead both within Helia and [within Kubo](https://github.com/ipfs/kubo/issues/10013) CI testing.
```
Making it more clear that this link does not just link to Kubo, but tests within Kubo.